tests/profiling/test_periodic.py

```diff
@@ -9,17 +9,22 @@
 def test_periodic():
     x = {"OK": False}
 
+    thread_started = threading.Event()
```

Comment:
hmm this approach isn't going to work for gevent 😕

Reply:
@jd any ideas on how to achieve thread synchronization here for gevent? Not sure how to do it between the main thread and the gevent thread. Tried queue and events to no avail.

Reply:
You can probably do it but you need the *real* `threading.Event` class: use `gevent.monkey.get_original("threading", "Event")` to get it.
Otherwise you'll access the gevent version of the `Event` class which won't work since it's for greenlets, not pthreads.

Reply:
Hmm I tried this and it didn't seem to work. I'll give it another try.